test/e2e/webdriver/driver.js

```diff
@@ -499,14 +499,13 @@ class Driver {
    * and returns a reference to the first matching element.
    *
    * @param {string | object} rawLocator - Element locator
-   * @param {number} timeout - Timeout in milliseconds
    * @returns {Promise<WebElement>} A promise that resolves to the found element.
    */
-  async findElement(rawLocator, timeout = this.timeout) {
+  async findElement(rawLocator) {
```

Comment:
why we have this change?

Reply:
Github messed it up when rebasing. Fixed